ssh/tailssh/incubator.go

```diff
@@ -170,6 +170,15 @@ func beIncubator(args []string) error {
 	}
 
 	euid := uint64(os.Geteuid())
+	if !*sftpMode && len(cmdArgs) == 1 && cmdArgs[0] == "-l" && euid == 0 {
+		// If we are trying to launch a login shell, just exec into login instead.
+		lp, err := exec.LookPath("login")
+		if err != nil {
+			return err
+		}
+		return unix.Exec(lp, []string{lp, "-f", *localUser}, subprocEnviron())
```

Comment:
`login` clears the environment by default, so subprocEnviron won't be applied.

You can pass `-p` to preserve the environment, but then see below re: additional environment sanitization.

Reply:
I initially passed in `nil` but that made my `TERM` variable not work. So maybe `login` does something more fancy?